pkg/datatransfers/data_transfers.go

```diff
@@ -138,21 +138,21 @@ func beginTransactions(ctx context.Context, f, t string) error {
 }
 
 func commitTransactions(ctx context.Context, f, t string, krid string, db *qdb.QDB) error {
```

Comment:
maybe we can tests this with unit tests also? 
We can create mock - implementation of qdb, which will return expected results on all operations